server/index.js

```diff
@@ -439,7 +440,7 @@ app.post('/get_client_urls', function (req, res) {
                 return;
             });
         };
-        
+
         verify_password(body.session, body.password, success, fail);
```

Comment:
Good question... I think that's fine, as long as all of the comparisons are done lowercase.